frontend/src/Chat/components/ChatList/ChatListModal.tsx

```diff
@@ -42,9 +42,10 @@ export const ChatListModal = (props: {
   const [isPublic, setIsPublic] = useState<boolean>(true)
 
   const handleCreateRoom = (): void => {
-    const requestData: ChatRoom = {
+    const requestData: ChatRoomReqDto = {
       name: newRoomName,
       created_by: props.user.id,
```

Comment:
created_byまだあとにいりますか？created_by_user_idと同じですか？

Reply:
同じです。。。
本来は必要ないのですが、TypeORMの制約で必要になる項目です。。。

こちらに詳細書きました！
https://github.com/katataku/ft_transcendence/issues/122

Reply:
Ohhhh, that's what it was about. I didn't read it yet because I didn't know exactly about typeorm.
 I'm starting to understand now.. Thank you!
あ～、そういうことだったんですね。まだtypeormのことをよく知らないから読まなかったんです。
少しずつ分かるようになっています。ありがとうございました！